de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/SarosSession.java

```diff
@@ -1150,6 +1181,9 @@ private SarosSession(final String id, IContainerContext context,
         userListHandler = sessionContainer
             .getComponent(UserInformationHandler.class);
 
+        referencePointManager = sessionContainer
+            .getComponent(IReferencePointManager.class);
+
         // ensure that the container uses caching
         assert sessionContainer.getComponent(ActivityHandler.class) == sessionContainer
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Correct one of the identical sub-expressions on both sides of operator "==" ([squid:S1764](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1764))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=257&issues=AWbeHvOCSgN4uCclRsG2&open=AWbeHvOCSgN4uCclRsG2)

[ISSUE_KEY]: AWbeHvOCSgN4uCclRsG2